component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/smsotp/SMSOTPAuthenticator.java

```diff
@@ -1658,4 +1669,114 @@ private long getUnlockTimeInMilliSeconds(AuthenticatedUser authenticatedUser) th
                     username, e);
         }
     }
+
+    private void publishPostSMSOTPGeneratedEvent(HttpServletRequest request, AuthenticationContext context)
```

Comment:
Shall we add method level comment here

Reply:
Fixed in 7d65437416ff6a2f9c995097fb0958fc95a58cf3